rag/llm/sequence2txt_model.py

```
@@ -44,6 +45,47 @@ def transcription(self, audio_path, **kwargs):
+            with tempfile.NamedTemporaryFile(suffix=".wav", delete=False) as f:
+                f.write(wav_data)
+                temp_path = f.name
+            try:
+                text, _ = self.transcription(temp_path)
+                if text.find("**ERROR**") >= 0:
+                    return False, text.replace("**ERROR**: ", "").strip()
+                return True, ""
+            finally:
+                if os.path.exists(temp_path):
+                    os.unlink(temp_path)
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)